index.html

```diff
@@ -15,233 +15,243 @@
     <header>
         <a href="#" class="logo"> Welcome </a>
 
-        <nav>
+        <div class="nav-menu">
+            <img src="./images/menu.svg" alt="menu-icon" class="menu nav-menu">
+        </div>
 
+        <nav class="mobile-menu">
+            <div class="close-menu">
+                <img src="./images/icons8-close.svg" alt="menu-close" class="close-menu-icon">
+            </div>
             <ul class="nav-list">
```

Comment:
- [ ] Please consider adding the horizontal line which is depicted in the given design, You are not required to implement the functionality at this point but to follow the design guideline we need to add the line. 

### Your design 
![image](https://github.com/ahmedeid6842/Portfolio/assets/69077061/9598b4fa-ce85-4c9a-a262-4f6e9c018b64)


### Figma design 
![image](https://github.com/ahmedeid6842/Portfolio/assets/69077061/0b5e65a8-1fb2-401e-a385-bedc48dab919)